_includes/header.html

```diff
@@ -8,7 +8,12 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="{{site.baseurl}}/assets/materialize/css/materialize.min.css"  media="screen,projection"/>
+      {% if jekyll.environment == "development" %}
+      <link href="{{site.baseurl}}/assets/materialize/css/materialize.min.css" rel="stylesheet" media="screen,projection">
+      {% endif %}
+      {% if jekyll.environment == "production" %}
+      <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" rel="stylesheet" media="screen,projection">
+      {% endif %}
```

Comment:
Please add `text/css`. If you have any specific reason for not adding let me know. :smile:

Reply:
Awesome. :wink: